# Temporary patch to apply to MXE's src/mdbtools.mk file while building docker
# image, until it's applied upstream at github's MXE repo.


diff --git a/src/mdbtools.mk b/src/mdbtools.mk
index 42d303c2..d9d0f557 100644
--- a/src/mdbtools.mk
+++ b/src/mdbtools.mk
@@ -1,19 +1,17 @@
 # This file is part of MXE. See LICENSE.md for licensing information.
 
 PKG             := mdbtools
-$(PKG)_WEBSITE  := https://sourceforge.net/projects/mdbtools/
+$(PKG)_WEBSITE  := https://github.com/mdbtools/mdbtools
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 0.7.1
-$(PKG)_CHECKSUM := 4eac1bce55066a38d9ea6c52a8e8ecc101b79afe75118ecc16852990472c4721
-$(PKG)_SUBDIR   := brianb-mdbtools-f8ce1cc
-$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
-$(PKG)_URL      := https://github.com/brianb/$(PKG)/tarball/$($(PKG)_VERSION)/$($(PKG)_FILE)
+$(PKG)_VERSION  := 1.0.0
+$(PKG)_CHECKSUM := 3446e1d71abdeb98d41e252777e67e1909b186496fda59f98f67032f7fbcd955
+$(PKG)_GH_CONF  := mdbtools/mdbtools/releases, v
 $(PKG)_DEPS     := cc glib
 
 define $(PKG)_UPDATE
-    $(WGET) -q -O- 'https://github.com/brianb/mdbtools/tags' | \
-    grep '<a href="/brianb/mdbtools/archive/' | \
-    $(SED) -n 's,.*href="/brianb/mdbtools/archive/\([0-9][^"_]*\)\.tar.*,\1,p' | \
+    $(WGET) -q -O- 'https://github.com/mdbtools/mdbtools/tags' | \
+    grep 'href="/mdbtools/mdbtools/archive/' | \
+    $(SED) -n 's,.*href="/mdbtools/mdbtools/archive/refs/tags/v\([0-9][^"_]*\)\.tar.*,\1,p' | \
     head -1
 endef
 
@@ -24,10 +22,11 @@ define $(PKG)_BUILD
         --build="`config.guess`" \
         --disable-shared \
         --disable-man \
+        --without-bash-completion-dir \
         --prefix='$(PREFIX)/$(TARGET)' \
         PKG_CONFIG='$(PREFIX)/bin/$(TARGET)-pkg-config'
-    $(MAKE) -C '$(1)' -j '$(JOBS)' install bin_PROGRAMS= sbin_PROGRAMS= noinst_PROGRAMS= html_DATA= || \
-    $(MAKE) -C '$(1)' -j 1 install bin_PROGRAMS= sbin_PROGRAMS= noinst_PROGRAMS= html_DATA=
+    $(MAKE) CFLAGS+='-Wno-deprecated-declarations' -C '$(1)' -j '$(JOBS)' install bin_PROGRAMS= sbin_PROGRAMS= noinst_PROGRAMS= html_DATA= || \
+    $(MAKE) CFLAGS+='-Wno-deprecated-declarations' -C '$(1)' -j 1 install bin_PROGRAMS= sbin_PROGRAMS= noinst_PROGRAMS= html_DATA=
 endef
 
 $(PKG)_BUILD_SHARED =
-- 
2.43.0

